ccmbioinfo/lfs
# CARDS-1741: Add a "PageStart" extension point to the patient UI

Also fixes [CARDS-1929](https://phenotips.atlassian.net/browse/CARDS-1929), [CARDS-1930](https://phenotips.atlassian.net/browse/CARDS-1930), [CARDS-1931](https://phenotips.atlassian.net/browse/CARDS-1931).

To test:
- Start yourexperience or cards in test mode, with --demo to activate the demo banner
- From Administration, enable the Downtime Warning to have two stacked banners
- Create a visit with surveys, then obtain the survey link as usual
- As patient, using the survey link, check that the banners are displayed as expected in the patient portal

This is a relatively LOW_RISK pr because of the reduced visibility of PageStart banners.

[CARDS-1929]: https://phenotips.atlassian.net/browse/CARDS-1929?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CARDS-1930]: https://phenotips.atlassian.net/browse/CARDS-1930?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CARDS-1931]: https://phenotips.atlassian.net/browse/CARDS-1931?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
